SimG4CMS/Calo/interface/HGCSD.h

```diff
@@ -11,6 +11,7 @@
 #include "SimG4Core/Notification/interface/BeginOfEvent.h"
 #include "SimG4CMS/Calo/interface/HGCNumberingScheme.h"
 #include "SimG4CMS/Calo/interface/HGCMouseBite.h"
+#include "Geometry/HGCalTBCommonData/interface/HGCalTBDDDConstants.h"
 
 #include <string>
 #include <TTree.h>
```

Comment:
@bsunanda , why this class depend on TTree?

Reply:
Because when we use plot debug we make a tree within the SD class

Reply:
@civanch @srimanob Please sign this PR